docs/cn_omf.md

```diff
@@ -176,6 +176,8 @@ WBTree Interior Node (one 4K page):
     | sequence number    |
     +--------------------+
 
+The `vblock idx` stored in kmd is an index within the vgroup specified by `vgroup id`.
+
```

Comment:
vblock index

Reply:
and throughout the document...

Also, capitalize "id" thoughout...